src/Ng/Scripts/Catalog2RegistrationV3-Reg3.cmd

```diff
@@ -0,0 +1,40 @@
+@echo OFF
+
+cd Ng
+
+:Top
+echo "Starting job - #{Jobs.catalog2registrationv3reg3.Title}"
+
+title #{Jobs.catalog2registrationv3reg3.Title}
+
+start /w ng.exe catalog2registration ^
+    -source #{Jobs.common.v3.Source} ^
+    -contentBaseAddress #{Jobs.catalog2registrationv3reg3.ContentBaseAddress} ^
+    -storageType azure ^
+    -instrumentationkey #{Jobs.common.v3.Logging.InstrumentationKey} ^
+    -vaultName #{Deployment.Azure.KeyVault.VaultName} ^
+    -clientId #{Deployment.Azure.KeyVault.ClientId} ^
+    -certificateThumbprint #{Deployment.Azure.KeyVault.CertificateThumbprint} ^
+    -verbose true ^
+    -interval #{Jobs.common.v3.Interval} ^
+    -storageBaseAddress #{Jobs.catalog2registrationv3reg3.StorageBaseAddress} ^
+    -storageAccountName #{Jobs.common.v3.c2r.StorageAccountName} ^
+    -storageKeyValue #{Jobs.common.v3.c2r.StorageAccountKey} ^
+    -storageContainer #{Jobs.catalog2registrationv3reg3.StorageContainer} ^
+    -useCompressedStorage #{Jobs.catalog2registrationv3reg3.UseCompressedStorage} ^
+    -compressedStorageBaseAddress #{Jobs.catalog2registrationv3reg3.StorageBaseAddressCompressed} ^
```

Comment:
It is the same structure as the registration2 script. I would let it the same for now and we can change it later.